bundle.yml

```diff
@@ -104,6 +104,10 @@ options :
         help  : Build the C version of CLOUDSC [ON|OFF]
         cmake : ENABLE_CLOUDSC_C={{value}}
 
+    - cloudsc-cuda :
+        help  : Build the CUDA C version of CLOUDSC [ON|OFF]
+        cmake : ENABLE_CLOUDSC_CUDA={{value}}
```

Comment:
Should we maybe add this to the `--with-cuda` option to turn these on? Or at least add a small comment to the README describing them and how to enable them? I realise that they hard-require Serialbox, so might not want to have them always on under `--with-cuda`.